Documentation/kubernetes-on-aws.md

```diff
@@ -178,6 +194,8 @@ If deploying a production Kubernetes cluster, consider establishing PKI independ
 
 #### Configure your DNS
 
+This step is not necessary if `createRecordSet` is true. In this case, a host record has already been created for you.
```

Comment:
Hmm, I guess I can see how this is separated between during the install vs post-install steps...